### ISSUE TITLE
a couple of issues when English and Arabic are used in a sentence

### ISSUE THREAD

**Issue by dannykoh** [Konash/arabic-support-unity#37]
Hi, I am making an app in Arabic language, and was encountered to a couple of problems regarding texts around line breaks and symbols when Arabic and English words are used at the same time in a sentence. Here goes...

1. problem with line breaks
![image](https://user-images.githubusercontent.com/49521060/71312470-b468f000-246e-11ea-9721-60ee520230dd.png)

When an English word exists in the middle of Arabic sentence and there is a line break because of the length of the sentence, there is a problem where the Arabic part that goes before the English word would get switched places with the part that goes after the English word. (Please check the screenshot I attached above) 
What should I do to fix this problem? I tried to look for previous similar issues reported in your github to no avail. 

2. problem with symbol ')' and '('
In the screenshot above, there are parentheses at both ends of the English words, Smart Sense and PAS. They are supposed to appear to be "(Smart Sense)" and "(PAS)", respectively, but, after I apply "LocalizationText.SetText() to them, the parentheses get reversed like in the screenshot. 
I had to make some revision to ArabicSupport.cs to make it work as it should, 
![image](https://user-images.githubusercontent.com/49521060/71312591-00686480-2470-11ea-8459-10a3985f279d.png)
but, I still feel that there should be the right way to fix this.

**Comment by danilavanilla** [comment]
@dannykoh did u fix this?

**Comment by dannykoh** [comment]
No, I haven't fixed the problem, yet. My client does not complain too much about the issue, so I'm leaving the issue as it is, but I still hope that the dev of Arabic Support plugin comes out and fixes it for good very soon.
